framework/scripts/cluster_control.py

```diff
@@ -34,6 +34,8 @@ def signal_handler(signal, frame):
     print("Error importing 'Wazuh' package.\n\n{0}\n".format(e))
     exit()
 
+import sys
```

Comment:
The `sys` module is already imported:

https://github.com/wazuh/wazuh/blob/4951ad978e5834c38b6a00f6317ba0cb75bc4041/framework/scripts/cluster_control.py#L7

Remove this import and fix these two lines:

https://github.com/wazuh/wazuh/blob/4951ad978e5834c38b6a00f6317ba0cb75bc4041/framework/scripts/cluster_control.py#L258-L259